mountainsort/src/core/msscriptcontroller.h

```diff
@@ -0,0 +1,31 @@
+/******************************************************
+** See the accompanying README and LICENSE files
+** Author(s): Jeremy Magland
+** Created: 4/19/2016
+*******************************************************/
+
+#ifndef MSSCRIPTCONTROLLER
+#define MSSCRIPTCONTROLLER
+
+#include <QObject>
+
+
+class MSScriptControllerPrivate;
+class MSScriptController : public QObject
+{
+    Q_OBJECT
+public:
+    friend class MSScriptControllerPrivate;
+    MSScriptController();
+    virtual ~MSScriptController();
+    Q_INVOKABLE QString fileChecksum(const QString &fname);
```

Comment:
What do you mean..... are you suggesting...
 Q_INVOKABLE QString fileChecksum(const QString &fname) const;
?

I guess it's really a static function. Should I make it static?